playbooks/rhv/inventory.example

```diff
@@ -0,0 +1,99 @@
+[all:vars]
```

Comment:
Based on https://docs.openshift.com/container-platform/3.10/install/prerequisites.html#install-config-network-using-firewalld, I think we may want to add the flag os_firewall_use_firewalld=true
@cwilkers

Reply:
That would be worth testing on (I've previously found bugs in load balancer configs WRT firewalling) but turning that on in the RHV example inventory contradicts our decision earlier in the thread to simplify the vars and inventory files.

Reply:
I don't see this flag in the https://github.com/openshift/openshift-ansible/blob/master/inventory/hosts.example ...
So either add it there or here no?